pkg/datapath/linux/ipsec/ipsec_linux_test.go

```diff
@@ -38,6 +38,10 @@ func setupIPSecSuitePrivileged(tb testing.TB) *slog.Logger {
 	return log
 }
 
+const (
+	ipsecDirBoth IPSecDir = IPSecDirIn | IPSecDirOut
```

Comment:
Lint is correct here. No need for this anymore, the concept of 'both' does not exist.

Reply:
Oops, forgot to remove it.